lib/metrics.ts

```diff
@@ -1,74 +1,83 @@
-'use strict';
+import promBundle from "express-prom-bundle";
+import { ILogger } from "nchat-dev-common";
 
-const promBundle = require('express-prom-bundle');
+import { PrometheusMetricsBackend } from "./backends/prometheus";
 
-const { PrometheusMetricsBackend } = require('./backends/prometheus');
+import { ExpressMiddlewareDefaultSettings, DefaultBackendSettings } from "./constants";
+import { IBackendSettings, IMetricsBackend } from "./types";
 
-const { ExpressMiddlewareDefaultSettings, DefaultBackendSettings } = require('./constants');
+export class Metrics {
```

Comment:
We can create an Interface for this class because it is exported in the index file.